encrypted-media/scripts/requestmediakeysystemaccess.js

```diff
@@ -0,0 +1,311 @@
+function runTest( config, qualifier ) {
+
+    var testname = testnamePrefix( qualifier, config.keysystem ) + ', requestMediaKeySystemAccess';
+
+    function expect_error(keySystem, configurations, expectedError, testname) {
+        promise_test(function(test) {
+            return navigator.requestMediaKeySystemAccess(keySystem, configurations).then(function(a) {
```

Comment:
Should it be [promise_rejects](https://github.com/w3c/testharness.js/blob/master/docs/api.md#promise-tests)?